config/user-commands.m4

```diff
@@ -96,12 +99,20 @@ dnl # These commands may take different command line arguments.
 dnl #
 AC_DEFUN([ZFS_AC_CONFIG_USER_COMMANDS_LINUX], [
 	AC_PATH_TOOL(BLOCKDEV, blockdev, "")
+	AC_PATH_TOOL(CHACL, chacl, "")
```

Comment:
Good point, but we still really should add this dependency.  It's fine if you want to keep it in a separate patch.